paper.tex

```diff
@@ -35,9 +36,9 @@
 \ccsdesc[500]{Computing methodologies~Parametric curve and surface models}
 
 % Re-enable these if/when published, also remove the `nonacm` version in \documentclass above.
-%\copyrightyear{2024}
-%\setcopyright{acmcopyright}
-%\conferenceinfo{Foo}{Month Day-Day, Year, Location}
+\copyrightyear{2024}
+\setcopyright{acmcopyright}
+%\conferenceinfo{High Performance Graphics}{July 26-28 2024, Denver}
```

Comment:
Should this be uncommented?

Reply:
That macro is not defined in the template, so uncommenting it causes an error. I believe the metadata is set correctly now, see eaac725.